### ISSUE TITLE
Missing Safari pin tab icon?

### ISSUE THREAD

**Issue by kingmatusevich** [sabnzbd/sabnzbd#1051]
Pinned tabs in any relatively modern version of Safari can display an SVG rendered icon in a colour of the website's choice. Sabnzbd still doesn't display one, and it's the kind of sites that would be traditionally pinned. It's such an easy win. 

You can see the documentation for adding it [here](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html).

Basically, it boils down to adding a `link` tag to the portal's html, like this:
```html
<link rel="mask-icon" href="website_icon.svg" color="red">
```

Here's a screenshot showing the "problem": 
<img width="197" alt="screen shot 2017-10-12 at 18 14 57" src="https://user-images.githubusercontent.com/1448511/31519716-a64c8138-af79-11e7-9d73-1ded3df5466e.png">

**Comment by Safihre** [comment]
You are in luck, it was added just a few days ago (https://github.com/sabnzbd/sabnzbd/pull/1045) and will be in the next 2.3.1 release!

**Comment by kingmatusevich** [comment]
Wow.
I've been waiting for it to be added for years, so this is a huge coincidence, thanks!

**Comment by Safihre** [comment]
I'm not a Mac user besides basic testing, so just didn't know about it 😄 ☁️

**Comment by kingmatusevich** [comment]
Makes sense! I don't know what took me so long to create an issue here, tbh. It's a small issue to have, but I'm so happy with Sabnzbd in general that this was the *one* thing I reeeeally wanted to happen.
